README.md

```diff
@@ -15,3 +16,8 @@ The agent manages container processes inside the VM, on behalf of the
 To enable agent debug output, add the `agent.log=debug` option to the guest kernel command line.
 
 See the [developer guide](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#enable-full-debug) for further details.
+
+## Developer mode
+
+Add `agent.devmode` to the guest kernel command line to enable
+[debug mode](#debug-mode) and allow the agent process to coredump (disabled by default).
```

Comment:
are you saying the above debug mode relies on this developer mode to work? In which case, maybe I'd put the two sections the other way around, or combine them. As it stands, I'm a bit confused about what each mode (debug and dev) gets you, and what their relationship is.

Reply:
I've reworded this slightly but the details are there if you squint ;) Debug mode enabled debug output (same for all Kata components), whereas developer mode enabled coredumping (but also implicitly enables debug mode to save you having to specify both options).

Reply:
... and before anyone comments, yes, I'm still updating this doc! ;)